connectors/src/lib/api/data_source_config.ts

```diff
@@ -1,6 +1,9 @@
 import type { ConnectorResource } from "@connectors/resources/connector_resource";
 import type { ConnectorModel } from "@connectors/resources/storage/models/connector_model";
-import type { DataSourceConfig } from "@connectors/types/data_source_config";
+import type {
+  DataSourceConfig,
+  DataSourceInfo,
+} from "@connectors/types/data_source_config";
 
 export function dataSourceConfigFromConnector(
```

Comment:
Would be nice to have those living on the `ConnectorResource`.

Reply:
this is broader refactor